hotTakeData.json

```diff
@@ -253,6 +277,14 @@
     "Ultimately, {thing} will succeed where others haven't, because it unites {thing} and {thing}.",
     "Every day when I wake up, I spent 30 minutes just reading the documentation for {language}. It sounds odd, but it always gets me excited to start working",
     "Why are {language} developers paid so much even though they don't understand {thing}?",
-    "The {problem} issue is just a thinly veiled advertisement for {thing}."
+    "The {problem} issue is just a thinly veiled advertisement for {thing}.",
+    "{person} debunked {technology} a long time ago.",
+    "{person} was the first true computer scientist.",
+    "{thing} fans are quickly taking over the tech world and that's good.",
+    "{thing} fans are quickly taking over the tech world and that's bad.",
+    "{technology} users couldn't even print Hello World in {language}.",
+    "0xDEADBEEF > {thing}",
+    "{problem} this, {problem} that. How about you learn {thing} instead?",
```

Comment:
I don't think this will capitalise, maybe worth putting something at the front?

Reply:
true, gonna think this through

Reply:
It does capitalise automatically